config/oidc_backends_config.xml.sample

```diff
@@ -100,4 +100,13 @@ Please mind `http` and `https`.
         <!-- (Optional) Override the default idp hint -->
         <!-- <idphint>cilogon</idphint> -->
     </provider>
+    
+    <! -- Documentation: https://docs.google.com/document/d/1vOyW4dLVozy7oQvINYxHheVaLvwNsvvghbiKTLg7RbY-->
```

Comment:
This link should probably go to https://galaxyproject.org/authnz/config/oidc/idps/elixir-aai/ assuming this is merged.

Reply:
Yes, the link should be updated as you suggest @bgruening . @kidanem could you update the doc link in the backends sample file? The Elixir AAI google link is now available within the galaxyproject.org docs.

Reply:
@kjellp - the link above doesn't work. Please provide a working link and I will update

Reply:
Link updated!